data/campaigns/Dead_Water/scenarios/10_The_Flaming_Sword.cfg

```diff
@@ -214,9 +214,9 @@
         [role]
             side=1
             race=merman
-            [filter_wml]
-                upkeep=full
-            [/filter_wml]
+            [not]
+                trait=loyal
+            [/not]
```

Comment:
That may match leaders too.
upkeep=loyal and upkeep=free might work, but it needs testing if these two are really the same. (at least I don't know)

Reply:
iirc, free is converted to loyal in the c++ code. ie, you won't see 'free' in the :inspect window for a unit, it will show as loyal. so it seems best to use upkeep=loyal instead of upkeep=free